### PR TITLE
Spanish translations

### PR DESCRIPTION
**Pull request by acrespo**
Hi ! I have translated the English files into Spanish language.

### CODE REVIEW

**Comment by UnickSoft** [comment]
Thank you so much.

**Comment by UnickSoft** [comment]
Thanks a lot.

File: lang/es/home.php [hunk 1]
```diff
@@ -0,0 +1,185 @@
+<?php
+
+    $g_lang["title_notg"] = "Encontrar camino más corto";
+    $g_lang["text"] = "<p>Crea un grafo y encuentra el camino más corto entre vértices. En la página de ayuda <a href=\"/en/help\">encontrarás un video tutorial</a>.</p>\n";
+    $g_lang["add_node"] = "Agregar vértice";
+    $g_lang["connect_nodes"] = "Conectr vértices";
```
Comment (mdb1) [inline lang/es/home.php:6]: Hey @acrespo you missed a letter here. I'll address it on another PR

Comment (mdb1) [reply lang/es/home.php:6]: https://github.com/UnickSoft/graphonline/pull/13

**Pull request merged**